package.json

```diff
@@ -194,7 +196,7 @@
     "execa": "^2.0.4",
     "form-data": "^2.5.1",
     "hat": "0.0.3",
-    "interface-ipfs-core": "^0.111.0",
+    "interface-ipfs-core": "https://github.com/ipfs/interface-js-ipfs-core/tarball/a810adfcbb977257851a4be3aae158a350119810/interface-js-ipfs-core.tar.gz",
```

Comment:
@alanshaw are you able to merge & release https://github.com/ipfs/interface-js-ipfs-core/pull/514, so this can be switched to real version ?